scripts/zones/Port_Jeuno/npcs/Shemo.lua

```diff
@@ -0,0 +1,67 @@
+-----------------------------------
+-- Area: Port Jeuno
+--  NPC: Shemo
+-----------------------------------
+local ID = require("scripts/zones/Port_Jeuno/IDs")
+require("scripts/globals/settings")
+require("scripts/globals/quests")
+require("scripts/globals/utils")
+-----------------------------------
+local entity = {}
+
+local beastman_seal = 0
+local kindred_seal = 1
+local kindred_crest = 2
+local high_kindred_crest = 3
+local sacred_kindred_crest = 4
+
+-- 0:Convert at 3: 1
+-- 1:Convert at 2: 1 (campaignAdditional Seal Battlefield Spoils Campaign)
+local isTradeNum2 = 0
+
+entity.onTrigger = function(player, npc)
+    local csid = 0x0160
```

Comment:
Is there a reason that csid is stored as hex here with no manipulation, and then referenced as decimal in onEventFinish?

Reply:
Unified csid to decimal numbers